swervelib/redux/HeliumCanCoderFactoryBuilder.java

```diff
@@ -43,24 +43,21 @@ private static class EncoderImplementation implements AbsoluteEncoder {
         private final Canandcoder encoder;
 
         private EncoderImplementation(Canandcoder encoder) {
-            this.encoder = encoder;
+            this.encoder = encoder; 
+            Settings settings = encoder.getSettings();
+            settings.setInvertDirection(true);
```

Comment:
could we please add a comment here that says we're inverting because that's the right thing to do with SDS Mk4i modules with the upside-down motors? In case we switch to something else in the future, just leave a breadcrumb here so people know where to look